crates/ruff/tests/integration_test.rs

```diff
@@ -2126,67 +2126,3 @@ unfixable = ["RUF"]
 
     Ok(())
 }
-
-#[test]
-fn verbose_show_failed_fix_errors() {
-    let mut cmd = RuffCheck::default()
```

Comment:
@dylwil3 I had to remove those tests. I think they did their job to proof that your change was working so I don't think we have to write new ones but if we do, then we should create a dummy rule instead to avoid depending on actual rule behavior.